source/io/module_eggShell.js

```diff
@@ -128,6 +130,22 @@
             return EggShell_GetArrayDimLength(v_userShell, vi, path, dim);
         };
 
+        Module.eggShell.resizeArray = publicAPI.eggShell.resizeArray = function (vi, path, newDimensionSizes) {
+            var int32Byte = 4;
+            var length = newDimensionSizes.length;
+            var buffer = Module._malloc(length * int32Byte);
```

Comment:
consider renaming length to rank and buffer to newLengths, to keep aligned with the names EggShell_ResizeArray is using